src/file-utils/edfi_lms_file_utils/file_reader.py

```diff
@@ -218,7 +244,9 @@ def _get_data_for_section(
         )
         return _default()
 
-    for _, section_id in sections[[c.SOURCE_SYSTEM_IDENTIFIER]].itertuples():
+    for _, section_id in sections[
+        [Keys.LMS_SECTION_SOURCE_SYSTEM_IDENTIFIER]
```

Comment:
This is the wrong key - it should be "SourceSystemIdentifier" here, not "LMSSectionSourceSystemIdentifier". I installed this latest version of the file_utils into the `record_counts.ipynb` notebook and ran into an error on this key.
```suggestion
        [Keys.SOURCE_SYSTEM_IDENTIFIER]
```

Reply:
I'll downgrade to the prior version and that should resolve the problem for me.

Reply:
Fixed